group_vars/rhcs.yml.sample

```diff
@@ -32,7 +32,7 @@ dummy:
 #  dev: 99
 
```

Comment:
Why are you modifying this file ?

Reply:
This file was generated by `generate_group_vars_sample.sh` as per "Submit a patch" in https://docs.ceph.com/ceph-ansible/master/dev/index.html.

After running the script, are we suppose to commit these changes?

Reply:
yes you're suppose to commit the changes generated but the `generate_group_vars_sample.sh` but your PR doesn't change anything for RHCS so it doesn't make sense that there's changes in rhcs.yml.sample file.

Reply:
Agreed

Reply:
We're thinking there are issues running `./generate_group_vars_sample.sh` on MacOS due to differences in `sed` behavior, as mentioned in https://unix.stackexchange.com/a/131940

Re-ran the generate script on a Ubuntu system and this file is no longer edited. Will update PR plus make further updates as per your comments. Thanks!